diff -Naur a/media/media_options.gni b/media/media_options.gni
--- a/media/media_options.gni	2021-10-29 05:07:20.000000000 +0200
+++ b/media/media_options.gni	2021-11-17 13:05:19.566276127 +0100
@@ -197,7 +197,7 @@
   ]
   _default_mojo_media_host = "gpu"
 } else if (is_chromeos_ash || is_mac || is_win ||
-           ((is_linux || is_chromeos_lacros) && use_vaapi)) {
+           ((is_linux || is_chromeos_lacros) && (use_vaapi || use_v4l2_codec))) {
   _default_mojo_media_services = [ "video_decoder" ]
   _default_mojo_media_host = "gpu"
 }
