diff -Naur chromium-89.0.4389.90.orig/media/media_options.gni chromium-89.0.4389.90/media/media_options.gni
--- chromium-89.0.4389.90.orig/media/media_options.gni	2021-03-13 00:57:26.000000000 +0100
+++ chromium-89.0.4389.90/media/media_options.gni	2021-04-02 10:15:11.716139785 +0200
@@ -206,7 +206,7 @@
     "video_decoder",
   ]
   _default_mojo_media_host = "gpu"
-} else if (is_chromeos_ash || is_mac || is_win || (is_linux && use_vaapi)) {
+} else if (is_chromeos_ash || is_mac || is_win || (is_linux && (use_vaapi || use_v4l2_codec))) {
   _default_mojo_media_services = [ "video_decoder" ]
   _default_mojo_media_host = "gpu"
 }
