--- a/ui/events/ozone/evdev/event_converter_evdev_impl.cc	2021-10-29 05:07:47.000000000 +0200
+++ b/ui/events/ozone/evdev/event_converter_evdev_impl.cc	2021-11-17 13:10:24.459047194 +0100
@@ -25,6 +25,10 @@
 const int kKeyReleaseValue = 0;
 const int kKeyRepeatValue = 2;
 
+#ifndef SW_PEN_INSERTED
+#define SW_PEN_INSERTED            0x0f  /* set = pen inserted */
+#endif
+
 // Values for the EV_SW code.
 const int kSwitchStylusInserted = SW_PEN_INSERTED;
 
