--- a/third_party/angle/gni/angle.gni	2021-10-29 05:09:19.000000000 +0200
+++ b/third_party/angle/gni/angle.gni	2021-11-17 13:24:58.840175762 +0100
@@ -132,7 +132,7 @@
   angle_enable_d3d9 = is_win && !angle_is_winuwp
   angle_enable_d3d11 = is_win
   angle_enable_gl =
-      (ozone_platform_gbm || !is_linux || (angle_use_x11 && !is_chromeos)) &&
+      ((ozone_platform_gbm || ozone_platform_wayland || ozone_platform_x11 || ozone_platform_drm) || !is_linux || (angle_use_x11 && !is_chromeos)) &&
       !is_fuchsia && !angle_is_winuwp && !is_ggp && !is_win_arm64
 
   angle_enable_vulkan =
