--- a/third_party/angle/gni/angle.gni	2021-07-19 20:47:19.000000000 +0200
+++ b/third_party/angle/gni/angle.gni	2021-08-17 14:00:07.577025644 +0200
@@ -120,7 +120,7 @@
   angle_enable_d3d9 = is_win && !angle_is_winuwp
   angle_enable_d3d11 = is_win
   angle_enable_gl =
-      (ozone_platform_gbm || !is_linux || (angle_use_x11 && !is_chromeos)) &&
+      ((ozone_platform_gbm || ozone_platform_wayland || ozone_platform_x11) || !is_linux || (angle_use_x11 && !is_chromeos)) &&
       !is_fuchsia && !angle_is_winuwp && !is_ggp && !is_win_arm64
 
   angle_enable_vulkan =
