--- a/chrome/common/chrome_paths.cc   2024-11-02 05:34:51.532944000 +0100
+++ b/chrome/common/chrome_paths.cc        2024-11-06 12:09:53.491750002 +0100
@@ -410,6 +410,15 @@
 
 #if BUILDFLAG(ENABLE_WIDEVINE)
     case chrome::DIR_BUNDLED_WIDEVINE_CDM:
+      base::PathService::Get(base::DIR_HOME, &cur);
+      cur = cur.Append(FILE_PATH_LITERAL(".local/lib"))
+               .AppendASCII(kWidevineCdmBaseDirectory);
+      if (base::PathExists(cur))
+        break;
+      cur = base::FilePath(FILE_PATH_LITERAL("/opt"));
+      cur = cur.Append(kWidevineCdmBaseDirectory);
+      if (base::PathExists(cur))
+        break;
       if (!GetComponentDirectory(&cur)) {
         return false;
       }
--- a/third_party/widevine/cdm/widevine_cdm_version.h 2024-11-02 05:36:10.626941700 +0100
+++ b/third_party/widevine/cdm/widevine_cdm_version.h      2024-11-06 12:07:38.851160668 +0100
@@ -11,5 +11,6 @@
 // If the Widevine CDM is available define the following:
 //  - WIDEVINE_CDM_VERSION_STRING (with the version of the CDM that's available
 //    as a string, e.g., "1.0.123.456").
+#define WIDEVINE_CDM_VERSION_STRING "undefined"
 
 #endif  // WIDEVINE_CDM_VERSION_H_
--- a/third_party/widevine/cdm/widevine.gni   2024-11-02 05:36:10.626941700 +0100
+++ b/third_party/widevine/cdm/widevine.gni        2024-11-06 12:11:24.304470026 +0100
@@ -27,7 +27,7 @@
 library_widevine_cdm_available =
     (is_chromeos &&
      (target_cpu == "x64" || target_cpu == "arm" || target_cpu == "arm64")) ||
-    (target_os == "linux" && target_cpu == "x64") ||
+    (target_os == "linux" && (target_cpu == "x64" || target_cpu == "arm" || target_cpu == "arm64")) ||
     (target_os == "mac" && (target_cpu == "x64" || target_cpu == "arm64")) ||
     (target_os == "win" &&
      (target_cpu == "x86" || target_cpu == "x64" || target_cpu == "arm64"))
@@ -44,7 +44,7 @@
 # component. See below.
 enable_widevine_cdm_component =
     enable_library_widevine_cdm &&
-    (is_win || is_mac || is_linux || is_chromeos_ash)
+    (is_win || is_mac || is_linux || is_chromeos_ash || is_chromeos)
 
 # Enable (Windows) Media Foundation Widevine CDM component.
 declare_args() {
